org/org.yaml

```diff
@@ -89,3 +89,4 @@ orgs:
     - waveywaves
     - withlin
     - wlynch
+    - dorismeixing
```

Comment:
Hey @dorismeixing! would you mind adding it so that the list remains sorted instead of adding it to the end of the list?
Otherwise, looks good to me!

Reply:
Thanks for the advice. I have updated it. Can you please check if it's in a right order now?